application/controllers/SuggestionsController.php

```diff
@@ -101,7 +101,8 @@ public function processAction()
                     ) {
                         yield [
                             'search' => $search,
-                            'label'  => $bpNode->getAlias() ?? $bpNode->getName(),
+                            'label'  => ($bpNode->getAlias() ?? $bpNode->getName())
+                                . " (" . implode("->", $bpNode->getPaths()[0]) . ")",
```

Comment:
1. Please use `' » '` enclosed with spaces.
2. Why the first path? There may be more. Why not one of the others?

Reply:
I tired a lot, and there where no otherpaths in this array. Do you have a test case where $bpNode->getPaths() returns more paths?

Reply:
Looking at the code, one case at least seems to be if a node is part of multiple parents.

Reply:
This is not resolved. @sukhwinder33445 Please look for cases where this might break or show incorrect information.